system/jlib/jstats.cpp

```diff
@@ -1835,7 +1902,8 @@ class CStatisticCollection : public CInterfaceOf<IStatisticCollection>
         if (parent)
         {
             parent->getFullScope(str);
-            str.append(':');
+            if (!str.isEmpty())
```

Comment:
I think this is because the top scope is global, and so blank.  I had another comment releated to this which I think is beyond the scope of this PR.

Reply:
parent->getFullScope returns an empty string when the scope is global.  Previously, the top level scope was workflow. 
 Now the top level scope may be global.  Global scope returns an empty string.  Without this, all scopes will be prefixed with a ':' character.